modelexpress_client/python/modelexpress/nixl_transfer.py

```
@@ -652,6 +652,96 @@ def receive_from_source(
+            return 0, 0, 0.0
+
+        mem = mem_type or self._accelerator_backend.nixl_mem_type
+        remote_descs = [(remote_addr, nbytes, dev) for (remote_addr, _local, nbytes, dev) in ranges]
+        local_descs = [(local_addr, nbytes, self._device_id) for (_remote, local_addr, nbytes, _dev) in ranges]
+
+        # Diagnostic: what we ask NIXL to READ from the remote. NIXL_ERR_NOT_FOUND
+        # at prep means these (addr,size,dev) aren't in a registered region of
+        # remote_agent_name as this agent knows it.
+        try:
+            _known = self._agent.check_remote_metadata(remote_agent_name)
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)